docs/src/pages/getting-started/_shared/tailwind-setup.mdx

```diff
@@ -0,0 +1,14 @@
+Wrap your Tailwind config with the `withUt` helper. You can learn more about our
+Tailwind helper in the ["Theming" page](/theming#with-tailwindcss)
+
+```tsx copy filename="tailwind.config.ts"
+import { withUt } from "uploadthing/tw";
+
+// Your existing Tailwind config
+const tailwindConfig = {
```

Comment:
This object is not typesafe if it lies out here.

Defining it inside the helper is and you get autocompletion etc